frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -644,15 +652,15 @@ export const Onboarding = () => {
                 </Text>
               </Box>
             )}
-            {isAdmin && isPaidPlan && (
+            {helpLink.visible && (
               <Box className={S.support} data-testid="help-section" p="lg">
                 <Stack spacing="xs">
                   <Title order={4}>{t`Need to talk with someone?`}</Title>
                   <Text>{t`Reach out to engineers who can help with technical troubleshooting. Not your typical support agents.`}</Text>
                 </Stack>
-                <Link to="mailto:help@metabase.com" key="help">
```

Comment:
suggestion, not a blocker:

If `helpLink.href` is simply `mailto:help@metabase.com`, I wonder if appending `?subject=...&body...` would be helpful

Reply:
But it's not `mailto:`
We're linking to two different versions of the help page, depending on the plan. This is why the helper is so useful.

The unit tests have these links in the assertions.